### ISSUE TITLE
Should verbal nouns be NOUN or VERB by part of speech?

### ISSUE THREAD

**Issue by ftyers** [UniversalDependencies/UD_Welsh-CCG#2]
Should _ladd_ in _Mi gafodd o ei __ladd__ ym Mangor_ be tagged as `NOUN` or `VERB` ?

**Comment by jheinecke** [comment]
It should be consistent with the marking used in other languages which also have verbnouns, so Irish in the first place. I am OK with both, but consistency is important. 
This applies for all cases with verbnouns (as for instance)

- declaratives _Mae Siôn yn caru Mari_ (John loves Mary), _Dwy newydd fywta afal_ (I have just eaten an apple)
- periphrastics with _gwneud_ (do): _mi wnes i weithio_ (I worked, lit: I did working)
- negative imperatives: _paid â bywta'r siocled_ (don't eat the chocolade)

**Comment by dan-zeman** [comment]
It won't be consistent with all languages that have verbal nouns because there are varying language-internal priorities, and UD explicitly allows languages to select `VERB` or `NOUN`. But agreed that Irish is the first candidate to look at, and to harmonize with.

**Comment by tsamardzic** [comment]
In the Serbian (Slavic) treebank, we tagged all deverbal nouns with `NOUN`, but we distinguished between "more verbal" and "more nominal" ones in how we annotated their dependants.  

I think a language-specific decision regarding the PoS tag should be taken based on how "verbal" a mixed category is in a given language. In some languages, such as Slavic, these mixed categories will tend to be on the nominal side (the term used in the literature is "opaque"). Other languages (such as English) will have some opaque and some transparent (more verbal) items. This Welsh  example looks to me more like Slavic than English.

**Comment by ftyers** [comment]
@tsamardzic are you referring to -ние type nouns, "писание" (_pisanje_ in `hbs` I guess). In this case I would say yes, they do look more like nouns,

* They can't take normal objects, e.g. _*писание книгу_ -- note unlike verbal adjectives which can.
* They can't take normal syntactic subjects (an agent can sometimes be expressed by `Ins` or `Gen` cases) 
* They can't take adverbial modification (either clausal or non-clausal)
* They can take adnominal modification (e.g. by verbal adjectives/participles)
* They do not have any implicit indication of TAMVE
* In order to be used predicatively a copula is needed.

I'm not sure how these criteria apply to Welsh (and other Celtic languages), but it would be good to find out.

**Comment by martinpopel** [comment]
I am not sure about Serbian, but in Czech

> They can't take normal objects, e.g. *писание книгу 

"psaní knihy" is possible and very common

> * They can't take normal syntactic subjects (an agent can sometimes be expressed
> by `Ins` or `Gen` cases)

Yes. We can also have agent expressed with possessive pronoun/adjective: "jeho/Petrovo psaní"

> * They can't take adverbial modification (either clausal or non-clausal)

We have "domácí(ADJ) psaní" vs. "psaní doma(ADV)" (the latter marks explicitly that the act of writing was done while being home, while the former could mean just that the writing has some property related to home).

PML-TQ `a-node [tag="NOUN", a-node [deprel="advmod"]]` query in UD-Czech finds also e.g. "vysílání automaticky(ADV)" (broadcasting automatically).

**Comment by tsamardzic** [comment]
@ftyers You are right, I meant primarily those such as _pisanje_ in `hbs` to which most of your tests apply (Serbian seems more strict than Czech). These are "more verbal" in the sense that they can have `obl` as a dependant, but still `NOUN`.  "More nominal" deverbal nouns are those not ending in -nje (e.g. _zapis_ 'record, something written').

**Comment by ftyers** [comment]
@martinpopel isn't _knihy_ genitive rather than accusative? (that's what I meant by "normal" object... although I should have been more clear). 

@jheinecke would the diagnostics I outline be useful in terms of Welsh ?

**Comment by martinpopel** [comment]
> isn't *knihy* genitive rather than accusative?

Yes, *knihy* is genitive. We cannot say **psaní knihu* with accusative, so this is a difference from the verb *psát* which can be used only with accusative, but not genitive.
However, Czech has [many verbs](http://ufal.mff.cuni.cz/vallex/3.0/#/filter/forms/direct_case/2) which need a genitive object (at least all Czech grammars I know call it "object"). In this case the object stays in genitive even after the verb is nominalized, e.g. *lituji té události* (I regret that event[case=gen].) and *litování té události* (regreting[NOUN] that event[case=gen]).

**Comment by jheinecke** [comment]
The criterium for a verboun is that the same argument appears syntactically different with respect to an inflected form. So for instant if the direct object is accusative with an inflected form and also accusative with the "infinitive", it's an infinitive, eg. German

- _Ich esse den Apfel_ (I eat the apple) with _den Apfel_ as accusative
- _den Apfel essen_ (to eat the appel), the object is still accusative (--> infintive)
- _das Essen des Apfels_ (the eating of the apple) `nmod `  has genitive cas (--> verb noun)

For Welsh
- _yfed cwrw_ ((to) drink beer): no mutation of the indefinite noun --> nmod of the verbnoun
- _mi yfodd gwrw_ he drank beer): soft mutation of the indefinite noun --> direct object
If the noun is definite, the mutation disappears (except, it fthe noun is feminine, but this is another case). I agree that the difference is not that much marked, since in many cases the mutation not there (and impersonal verbs do not have soft mutation neither. 

So does the case of the "genitive object" changes in Czech if the verb is in an infinite form ?

**Comment by amir-zeldes** [comment]
I think I mentioned this in another issue - this is pretty rare in Slavic, but there are also examples with accusative, e.g. Church Slavonic:

```
о събрании съборъ - about assembly.LOC council.ACC 'about assembling the council' 
```

"Assembly" is etymologically the same type of verbal noun in -nie, but can take accusative objects.

In Hebrew, by contrast, accusative objects are totally fine for verbal nouns with 'possessive' agents, though it is a little fancy:

```
shmi'at.i        et   ha-shir - my hearing the song
hearing.POSS1SG  ACC  DEF-song
```

**Comment by martinpopel** [comment]
> So does the case of the "genitive object" changes in Czech if the verb is in an infinite form ?

No. The object stays always in genitive: *lituji té události* (I regret the event[gen]), *litovat té události* (to regret the event[gen]), *litování té události* (regreting[NOUN] the event[gen]).
I was not able to find accusative objects (without any preposition) of deverbal nouns in Czech. (E.g. **shromáždění sbor[acc]* is incorrect in Czech, the only possibility is *shromáždění sboru[gen]* (assembling of the council).)

**Comment by jheinecke** [comment]
I think in German genitive objects do not change with deverbal nouns neither. The problem is, there are so few, and the example I think of sounds too artficial (_er gedenkt der Helden_). I am even unsure whether the term `verbnoun `is employed at all in German syntax.

**Comment by dan-zeman** [comment]
> I am even unsure whether the term verbnoun is employed at all in German syntax.

At least it [used to be](https://books.google.cz/books?id=TNRIAAAAcAAJ&pg=PA203&lpg=PA203&dq=Verbalnomen+deutsche+Grammatik&source=bl&ots=jHTszYug5X&sig=e0IWfJzMW8oRt5NOr0WyW0HKmiw&hl=cs&sa=X&ved=0ahUKEwjvu-DWyJ7bAhVDLewKHc2OAQsQ6AEITzAF#v=onepage&q=Verbalnomen%20deutsche%20Grammatik&f=false) :-)

**Comment by savary** [comment]
This discussion is important for the PARSEME [annotation guidelines](http://parsemefr.lif.univ-mrs.fr/parseme-st-guidelines/1.1/?page=010_Definitions_and_scope/030_Syntactic_variants_of_VMWEs#non-verbal-variants) on verbal multiword expressions (VMWEs). We try to capture the fact that **regular** nominalizations of VMWEs like (PL) _podejmowanie decyzji_ 'the-making decision.GEN'  should be annotated as variants of canonical forms like _podejmować decyzę_ 'make decision.ACC'. Conversely, lexicalized (**non-systematic**) nominalizations like (FR) _la prise de décision_ 'the take.PASTPART.FEM of decision' are nominal rather than verbal MWEs.

My main point is that deciding if _podejmowanie_ 'taking' is a NOUN or a VERB straightforwardly follows from which lemma we choose for this word. If we choose
     <tt> 3 podejmowanie **podejmować VERB** .....</tt>
then this is a verb. But if we choose
     <tt> 3 podejmowanie **podejmowanie NOUN** .....</tt>
then it is a noun. Neither solution is satisfactory since in the first case we have a verb that syntactically behaves like a noun. In the latter we lose the information that _podejmowanie_ 'making' derives from _podejmować_ 'make' in a **morphologically regular** way.

Note that in English UD the former solution is chosen for many cases:
    <tt>#defended **making** the December decision</tt>
    <tt>14   making  **make**  **VERB**  VBG VerbForm=Ger    13  xcomp   13:xcomp    _ </tt>
    <tt>...</tt>
    <tt>#I don't recall **making** any bad faith claims</tt>
    <tt>12   making  **make**  **VERB**  VBG VerbForm=Ger    11  xcomp   11:xcomp    _</tt>

To solve this, a double category, and a double lemma (?), would probably be needed. A double category can probably be achieved with the UD [VerbForm](http://universaldependencies.org/u/feat/VerbForm.html) feature. A double lemma is probably out of UD scope. (Not sure, I'm relatively new to UDs....)

PS: In Polish the "flexemic" tagset by  ([Przepiórkowski & Woliński 2003](http://nlp.ipipan.waw.pl/~adamp/Papers/2003-eacl-ws12/ws12.pdf))
 addresses this double nature of forms whose POS changes due to regular inflection (while inflection is actually expected to keep the POS intact).
